src/utilities/filetypes/WorkflowJSON.cpp

```diff
@@ -377,7 +377,7 @@ namespace detail {
   openstudio::path WorkflowJSON_Impl::rootDir() const {
     Json::Value defaultValue(".");
     Json::Value root = m_value.get("root", defaultValue);
-    return toPath(root.asString());
+    return toPath(root.asString()).remove_trailing_separator();
```

Comment:
Boost filesystem in 1.78 is now V4, and it follows the C++17 STL std::filesystem (which is worse), and doesn't treat a directory with a trailing separator the same way

cf https://www.boost.org/doc/libs/1_78_0/libs/filesystem/doc/v4.html
2efc0d

ref c0248cf8f27af8510711e4397c2e867f4d904e91